main.go

```diff
@@ -449,7 +449,7 @@ func main() {
 		if !installed {
 			log.Printf("Installing")
 
-			cmds, err := rubycommand.GemInstall("cocoapods", useCocoapodsVersionFromPodfileLock)
+			cmds, err := rubycommand.GemInstall("cocoapods", useCocoapodsVersionFromPodfileLock, false)
```

Comment:
It seems that the API has changed over time. [Here](https://github.com/bitrise-io/go-utils/commit/0c47c16813a4209321c9d0dba2e75867d6901ff2#diff-adcd7816a80eef9ca3c99dcbdf70460e5a26532a47ae53b458d8f769a9e3570aL156) is the exact change.
The `enablePrerelease` parameter appends the `--prerelease` flag to the `gem install` command if enabled.